core/lib/protobuf_config/src/proto/config/general.proto

```diff
@@ -20,28 +20,29 @@ import "zksync/config/pruning.proto";
 import "zksync/config/object_store.proto";
 
 message GeneralConfig {
-  optional config.database.Postgres postgres = 1;
-  optional config.api.Api api = 2;
-  optional config.contract_verifier.ContractVerifier contract_verifier = 3;
-  optional config.circuit_breaker.CircuitBreaker circuit_breaker = 5;
-  optional config.chain.Mempool mempool = 6;
-  optional config.chain.OperationsManager operations_manager = 8;
-  optional config.chain.StateKeeper state_keeper = 9;
-  optional config.house_keeper.HouseKeeper house_keeper = 10;
-  optional config.prover.Prover prover = 12;
-  optional config.utils.Prometheus prometheus = 15;
-  optional config.database.DB db = 20;
-  optional config.eth.ETH eth = 22;
-  optional config.prover.WitnessGenerator witness_generator = 24;
-  optional config.prover.WitnessVectorGenerator witness_vector_generator = 25;
-  optional config.prover.ProofCompressor proof_compressor = 27;
-  optional config.prover.ProofDataHandler data_handler = 28;
-  optional config.prover.ProverGroup prover_group = 29;
-  optional config.prover.ProverGateway prover_gateway = 30;
-  optional config.snapshot_creator.SnapshotsCreator snapshot_creator = 31;
-  optional config.observability.Observability observability = 32;
-  optional config.vm_runner.ProtectiveReadsWriter protective_reads_writer = 33;
-  optional config.object_store.ObjectStore core_object_store = 34;
+    optional config.database.Postgres postgres = 1;
+    optional config.api.Api api = 2;
+    optional config.contract_verifier.ContractVerifier contract_verifier = 3;
+    optional config.circuit_breaker.CircuitBreaker circuit_breaker = 5;
+    optional config.chain.Mempool mempool = 6;
+    optional config.chain.OperationsManager operations_manager = 8;
+    optional config.chain.StateKeeper state_keeper = 9;
+    optional config.house_keeper.HouseKeeper house_keeper = 10;
+    optional config.prover.Prover prover = 12;
+    optional config.utils.Prometheus prometheus = 15;
+    optional config.database.DB db = 20;
+    optional config.eth.ETH eth = 22;
+    optional config.prover.WitnessGenerator witness_generator = 24;
+    optional config.prover.WitnessVectorGenerator witness_vector_generator = 25;
+    optional config.prover.ProofCompressor proof_compressor = 27;
+    optional config.prover.ProofDataHandler data_handler = 28;
+    optional config.prover.ProverGroup prover_group = 29;
+    optional config.prover.ProverGateway prover_gateway = 30;
+    optional config.snapshot_creator.SnapshotsCreator snapshot_creator = 31;
+    optional config.observability.Observability observability = 32;
+    optional config.vm_runner.ProtectiveReadsWriter protective_reads_writer = 33;
+    optional config.vm_runner.BasicWitnessInputProducer basic_witness_input_producer = 38;
```

Comment:
nit: Honestly I would put this at the end as non-sequential numbers might confuse somebody. Also, you partially re-formatted this file